src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -103,7 +110,9 @@
     }
   }
 
-  function MdFabSpeedDialFlingAnimation() {
+  function MdFabSpeedDialFlingAnimation($timeout) {
+    function delayDone(done) { $timeout(done, cssAnimationDuration, false); }
+
     function runAnimation(element) {
```

Comment:
Why are you manually setting the style animation properties?
Why not use `$animateCSS( )`